scripts/sanitycheck

```diff
@@ -1013,11 +1013,24 @@ def main():
         suite.load_from_file(last_run, filter_status=['skipped', 'passed'])
         suite.selected_platforms = set(p.platform.name for p in suite.instances.values())
     elif options.load_tests:
-        suite.load_from_file(options.load_tests, filter_status=['skipped'])
+        suite.load_from_file(options.load_tests, filter_status=['skipped', 'error'])
```

Comment:
why skipping 'error' is introduced here?

Reply:
we cant test/run something that failed, did not build